Gopkg.lock

```diff
@@ -2007,7 +2007,7 @@
   revision = "b3a7cee44a305be0a69e1b9ac03018307287e1b0"
 
 [[projects]]
-  digest = "1:f27698f7ae7864893ebcfb843e44d821263ac1dcf0ba1d5c2353f9d319a2f28d"
```

Comment:
I had the same update locally and I discarded it. Do you know why it happens?

Reply:
no didn't noticed it before.
maybe due to the golang version